pkg/volume/quobyte/quobyte_util.go

```diff
@@ -46,3 +93,53 @@ func (mounter *quobyteMounter) pluginDirIsMounted(pluginDir string) (bool, error
 func (mounter *quobyteMounter) correctTraillingSlash(regStr string) string {
 	return path.Clean(regStr) + "/"
 }
+
+func validateRegistry(registry string) bool {
+	if len(registry) == 0 {
+		return false
+	}
+
+	for _, hostPortPair := range strings.Split(registry, ",") {
+		if _, _, err := net.SplitHostPort(hostPortPair); err != nil {
+			return false
+		}
+	}
+
+	return true
+}
+
+// ADD Annotations to a PersistentVolume all information needed to delete the Volume
+func addVolumeAnnotations(apiServer, adminSecret, adminSecretNamespace string, pv *api.PersistentVolume) {
```

Comment:
@rootfs @jsafrane what about adding a generic method `func addVolumeAnnotations(params map[string]string, pv *api.PersistentVolume)` to the `volume/util`, similar to https://github.com/kubernetes/kubernetes/pull/31251/files#diff-c3acf7a7fec401d0a3cb4998088ee1e5R114 ? The same for `parseVolumeAnnotations` (`func parseVolumeAnnotations(pv *api.PersistentVolume) (map[string]string, error)`)? This would remove the need that every Volume Plugin writes it own methods for Annotations adding/parsing. 

If you think this would be an improvement I would add this change in this PR (or an separated if you want).